internal/pkg/cli/flag.go

```diff
@@ -52,6 +52,9 @@ const (
 	deleteSecretFlag      = "delete-secret"
 	svcPortFlag           = "port"
 
+	noSubscriptionFlag = "no-subscriptions"
+	subscribeFlag      = "subscribe"
```

Comment:
To be consistent with the two flag names:
```suggestion
	noSubscriptionFlag = "no-subscribe"
	subscribeFlag      = "subscribe"
```
Also dang, I kinda wish we had named our flags `skip-*` but I see that we have a precedent for `no-*`